libs/librepcb/projecteditor/schematiceditor/schematiceditor.cpp

```diff
@@ -696,31 +696,51 @@ void SchematicEditor::renameSchematic(int index) noexcept {
   }
 }
 
-QStringList SchematicEditor::getSearchToolBarCompleterList() noexcept {
-  QStringList list;
+QList<SI_Symbol*> SchematicEditor::getSearchCandidates() noexcept {
+  QList<SI_Symbol*> candidates = {};
   foreach (const Schematic* schematic, mProject.getSchematics()) {
-    foreach (const SI_Symbol* symbol, schematic->getSymbols()) {
-      list.append(symbol->getName());
+    Q_ASSERT(schematic);
+    foreach (SI_Symbol* symbol, schematic->getSymbols()) {
+      candidates.append(symbol);
     }
```

Comment:
Could be simplified:

```suggestion
    candidates += schematic->getSymbols();
```